package.json

```diff
@@ -246,7 +246,7 @@
     "mini-css-extract-plugin": "~2.4.7",
     "minimatch": "9.0.3",
     "next-sitemap": "^3.1.10",
-    "ng-packagr": "~19.0.0",
+    "ng-packagr": "~19.1.0-next.3",
```

Comment:
fyi this is at a different version. I know you'll change it to the stable in this PR though.